D/DSDP/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "DSDP"
+version = v"0.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://www.mcs.anl.gov/hs/software/DSDP/DSDP5.8.zip", "8915e55456f1a7cc5c970ad157d094a5fb399737cf192dfe79b89c2d94d97a8a"),
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd DSDP*
+export DSDPROOT=${PWD}
+
+make -j${nproc} LAPACKBLAS="-L${libdir} -lopenblas -lm" dsdpapi
+make -j${nproc} oshared
+
+mv lib/* $libdir
+mv include/dsdp* $includedir
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = Product[
+]
```

Comment:
No products?

Reply:
oh :man_facepalming: forgot to re-add it